default.nix

```diff
@@ -11,13 +11,27 @@ let
     "^stack.*\.yaml$"
     "^test.*$"
   ];
-  mkMonadBayes = stackYaml: pkgs.haskell-nix.stackProject {
-    inherit stackYaml;
-    src = pkgs.lib.sourceByRegex ./. source;
-  };
-  monad-bayes-ghc84 = mkMonadBayes "stack-ghc844.yaml";
-  monad-bayes-ghc86 = mkMonadBayes "stack-ghc865.yaml";
-  monad-bayes-ghc88 = mkMonadBayes "stack-ghc881.yaml";
+  mkMonadBayes = stackYaml: doBench:
```

Comment:
Nit: `doBench` suggests that the benchmarks are run, but this only builds them, right? How about `buildBench` instead?

Reply:
This is just the conventional name for this. It's called `doBenchmark` in vanilla nix I think. It is silly to actually run benchmarks in nix build anyway.

Reply:
Cool, I rectract my nit then :)